draft-ietf-quic-transport.md

```diff
@@ -3896,6 +3890,15 @@ secure key, the attacker will not be able to generate forward-secure protected
 packets with ACK frames.
 
 
+## Opportunistic ACK Attack
+
+An endpoint that acknowledges packets it has not received might cause a
+congestion controller to permit sending at rates beyond what the network
+supports.  An endpoint MAY skip packet numbers when sending packets to detect
+this.  An endpoint can then drop connection state and stop sending packets
+toward the misbehaving peer.
```

Comment:
That just means closing the connection, doesn't it? Why don't we say it explicitly?

Reply:
Because "closing" generally means generating an error code, etc. to inform a peer that it made a mistake.  This is malicious behavior.  Also, by not making it normative, we leave room for any implementation-specific responses.